tests/drivers/adc/adc_api/src/test_adc.c

```diff
@@ -101,6 +101,16 @@
 #define ADC_ACQUISITION_TIME	ADC_ACQ_TIME_DEFAULT
 #define ADC_1ST_CHANNEL_ID	14
 
+#elif defined(CONFIG_BOARD_TLSR9518ADK80D)
+
+#define ADC_DEVICE_NAME		DT_LABEL(DT_INST(0, telink_b91_adc))
```

Comment:
```suggestion
#define ADC_DEVICE_NODE		DT_INST(0, telink_b91_adc)
```
See 7022eb05e4b68d25643e1462afcbb694a4f57bdb.

Reply:
Done.